CanalTP/NmmPortalBundle
# Remove useless navitia.io feature

# Description

This PR removes the navitia.io feature which is completely useless.
This is related to CanalTP/SamCoreBundle#40 which needs to be merged first.

## How to test

Here are the following steps to test this pull request:

- Update both SamCoreBundle and NmmPortalBundle 
- navitia.io tab must have disappeared
- The rest of the features should continue to work correctly

## Team reviewers (optional)

NMM guilde
